.nvmrc

```diff
@@ -1 +0,0 @@
-12
```

Comment:
This file is actually quite useful when using nvm/fnm/similar. Perhaps we should only change the version contained in this file?

Reply:
k, will add it back with node 16